### ISSUE TITLE
Not taking care of .jpeg etc

### ISSUE THREAD

**Issue by prats226** [prajnak/img_preprocess#3]
https://github.com/prajnak/img_preprocess/blob/4a2f843117d47531c4815c1eb8a196c90e29a733/preprocess.py#L30

**Comment by prajnak** [comment]
yeah, this is kind of a hack. I can get rid of this entire block where I store image format as I later use the `imghdr` library which uses file format reading to figure out an image format irrespective of file extension.
